nep-dapi.mediawiki

```diff
@@ -797,6 +797,90 @@ Example
 </pre>
 
 
+====verifyMessage====
+Returns whether the provided signature data matches the provided message and was signed by the account of the provided public key.
+
+
+'''Method Interface'''
+
+<pre>
+function verifyMessage({
+  message: string,
+  data: string,
+  publicKey: string,
+}): Promise<{result: boolean}>
+</pre>
+
+
+'''Input arguments'''
+
+<pre>
+interface VerifyMessageArgs {
+  message: string; // Salt prefix + original message
+  data: string; // Signed message
+  publicKey: string; // Public key of account that signed message
```

Comment:
does the app need to pass the public key in? can the provider just use a public key of an active wallet to verify the message?

Reply:
The intent of the method was more as a utility for verifying any message for any account. Similar to how the `getBalance` method can be used to query the balance of any account. Not just the one connected to app.